.changeset/long-jobs-argue.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/clickhouse': patch
+---
+
+Added the new `mastra_channel_state` table to the internal table registry for compatibility with the latest `@mastra/core`.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)